2022-01-12 12:14:57,569 INFO: Initialized curve data.
2022-01-12 12:14:57,572 INFO: Setting up initial twist data...
2022-01-12 12:14:59,116 INFO: Twist setup complete.
2022-01-12 12:14:59,116 INFO: Searching for rational points on each twist...
2022-01-12 12:15:02,418 INFO: Finished searching for rational points on each twist.
2022-01-12 12:15:02,419 INFO: Testing local solvability of twists...
2022-01-12 12:35:27,054 ERROR: An exception occurred. Recording data and exiting.
Traceback (most recent call last):
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 464, in <module>
    loc_solv = twists_locally_solvable(curve, twist_index=i)
  File "/project/arithgeo/drhast/twocover-descent/./twocover-processor.sage.py", line 245, in twists_locally_solvable
    ls2 = magma.function_call("is_genus5_locally_solvable_at_2", Z)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1162, in function_call
    return self._do_call(fun, nvals)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 1212, in _do_call
    return self(code)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 791, in __call__
    A = Expect.__call__(self, x)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/interface.py", line 295, in __call__
    return cls(self, x, name=name)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/expect.py", line 1468, in __init__
    self._name = parent._create(value, name=name)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/interface.py", line 500, in _create
    self.set(name, value)
  File "/share/pkg.7/sagemath/9.3/install/lib/python3.9/site-packages/sage/interfaces/magma.py", line 628, in set
    raise TypeError("Error executing Magma code:\n%s" % out)
TypeError: Error executing Magma code:

Current total memory usage: 16373.8MB, failed memory request: 32.0MB
System Error: User memory limit has been reached
2022-01-14 11:32:12,753 INFO: Loaded curve data from file.
2022-01-14 11:32:12,753 INFO: Reducing MW generators...
2022-01-14 11:32:12,753 INFO: Finished reducing MW generators.
2022-01-14 11:32:12,755 INFO: Skipped Chabauty because no base point (delta = ['-64', '65', '47', '-49', '65', '64'])
2022-01-14 11:32:12,755 INFO: Skipped Chabauty because no base point (delta = ['-1', '1', '1'])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because MW generators haven't been computed. (delta = ['1'], g = [-4, 4, 3, -3, 4, 4])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because no base point (delta = ['9', '-9', '-8', '7', '-7', '-8'])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because MW generators haven't been computed. (delta = ['-65', '65', '199/4', '-97/2', '251/4', '64'], g = [-4, 4, 3, -3, 4, 4])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because MW generators haven't been computed. (delta = ['7', '-8', '-5', '6', '-8', '-8'], g = [-4, 4, 3, -3, 4, 4])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because no base point (delta = ['4', '-3', '-5', '3', '-3', '-4'])
2022-01-14 11:32:12,756 INFO: Skipped Chabauty because MW generators haven't been computed. (delta = ['4', '-5', '-2', '3', '-4', '-4'], g = [-4, 4, 3, -3, 4, 4])
2022-01-14 11:32:12,836 INFO: Tasks complete. Recording data and exiting.
2022-02-12 17:20:05,474 INFO: Loaded curve data from file.
2022-02-12 17:20:05,475 INFO: Tasks complete. Recording data and exiting.
